js/settings.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function() {
```

Comment:
jQuery is deprecated, are you familiar with Vue?

Otherwise I can also have a look to move this JS code to a vue based one

Reply:
I was not familiar with Vue, I guess I am now ! I didn't realize the code I was looking at for reference in Activities was being replaced but I saw there's a pull request in progress.
Anyway I managed to move the settings to Vue (after quite some trouble, being a noob !).